src/components/NotificationCenter/index.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { values } from "lodash";
+import { connect } from "react-redux";
+import Notification from "./Notification";
+import TodoList from "../TodoList";
+
+
+export function NotificationCenter ({notifications, dismissNotification, deactivateNotification}) {
+  return (
+    <div className="notification-center">
+      <ul>
+        {notifications.map(notification => {
+          return (
+            <li key={notification.id}>
+              <Notification
+                notification={notification}
+                onDismiss={() => dismissNotification(notification.id)}
+                onDeactivate={() => deactivateNotification(notification.id, notification.todo.id)}
```

Comment:
Here you call the function `deactivateNotification` with two arguments -> `deactivateNotification(notification.id, notification.todo.id)`.